### ISSUE TITLE
README needs to document what file format coverage is expected in, and how the search for coverage files works

### ISSUE THREAD

**Issue by dabrahams** [codecov/codecov-action#1310]
My system generates .json files by default.  They can be converted to XML, which appears in the README examples but it's not clear whether that's a requirement of your system or not.  It's also not clear where the plugin will search for these coverage files by default.

**Comment by mjankowski** [comment]
Agreed this would be an improvement ... the readme documents that `files` is an optional argument -- so presumably there is some default rules/pattern here -- but doesn't say what the default selection will be. I'm guessing it's something like `*coverage*xml` or similar pattern?

**Comment by dabrahams** [comment]
You're asking me?  I filed this because I don't know(!)

**Comment by mjankowski** [comment]
Please accept my apologies. I'm not asking you, I'm joining you in the increasingly vocal chorus of voices who don't know what files are selected by default and would love to be told.

**Comment by rohan-at-sentry** [comment]
@mjankowski @dabrahams 

JSON coverage files are supported also. [A full list of supported coverage report formats can be found here](https://docs.codecov.com/docs/supported-report-formats)

[Default file patterns can be found here ](https://docs.codecov.com/docs/path-regexp#regexp-patterns)

**Comment by mjankowski** [comment]
Thanks for response!

> [Default file patterns can be found here ](https://docs.codecov.com/docs/path-regexp#regexp-patterns)

...this seems to show how supplied paths would be regexp-processed ... but not what the default search locations for files/dirs is?
